src/themes/teams/components/Header/headerDescriptionStyles.ts

```diff
@@ -1,11 +1,17 @@
+import * as _ from 'lodash'
+
 import { pxToRem } from '../../utils'
-import { ICSSInJSStyle } from '../../../types'
+import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
+import { HeaderProps } from '../../../../components/Header/Header'
+import { HeaderVariables } from './headerVariables'
 
-export default {
-  root: ({ variables: v }): ICSSInJSStyle => ({
+const headerStyles: ComponentSlotStylesInput<HeaderProps, HeaderVariables> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
     display: 'block',
+    color: _.get(v.colors, p.color, v.color),
```

Comment:
I don't know what we can do there, but this line looks very confusing 🤔

Reply:
agreed, but not sure what we can do either.. 😢